core/src/packet_sender_info.rs

```diff
@@ -0,0 +1,56 @@
+/// struct PacketSenderInfo contains sender information for pakcets in baning_stage's
+/// unprocessed_buffer. It provides metrics for stake weight block packing prioritization.
```

Comment:
nit: don't want to use specific variables from other modules, `unprocessed_buffer` -> `unprocessed queue`